tests/test_pipeline.py

```diff
@@ -145,7 +145,9 @@ def test_ensure_started_failure_to_start(client, pipeline_endpoint_behavior_mock
 
     with pytest.raises(Exception) as ex:
         pipeline.ensure_started()
-        assert error_message in str(ex.value)
+
+    # the assert needs to be on this level
```

Comment:
Remove comment - it explains the change but is not helpful for the next person reading the code because it can only be understood when comparing these lines with current code.

Reply:
k